sllurp/llrp_proto.py

```diff
@@ -2911,6 +2911,10 @@ def __init__(self, llrpcli, msgid, priority=0, state='Disabled',
         if state not in ROSpecState_Name2Type:
             raise LLRPError('invalid ROSpec state {} (need [{}])'.format(
                             state, ','.join(ROSpecState_Name2Type.keys())))
+        for antid in tx_power:
```

Comment:
I don't understand this one, tx_power map will be quite small (up to 32 antennas, usually just 4)
Maybe I'm missing something but it will no make any difference using keys().
BTW, iterkeys() is not available in Python3.